src/lib/components/data-encoder-status.svelte

```diff
@@ -6,16 +6,30 @@
     faLongArrowAltUp,
   } from '@fortawesome/free-solid-svg-icons';
 
+  import {
+    dataEncoderEndpoint,
+    lastDataEncoderStatus,
+  } from '$lib/stores/data-encoder-config';
+
   import {
     dataConverterPort,
     lastDataConverterStatus,
   } from '$lib/stores/data-converter-config';
 
   import Tooltip from '$lib/components/tooltip.svelte';
+  $: notRequested = $dataConverterPort
+    ? $lastDataConverterStatus === 'notRequested'
+    : $lastDataEncoderStatus === 'notRequested';
+  $: error = $dataConverterPort
+    ? $lastDataConverterStatus === 'error'
+    : $lastDataEncoderStatus === 'error';
+  $: success = $dataConverterPort
+    ? $lastDataConverterStatus === 'success'
+    : $lastDataEncoderStatus === 'success';
 </script>
 
-{#if $dataConverterPort}
-  {#if $lastDataConverterStatus === 'notRequested'}
+{#if $dataEncoderEndpoint || $dataConverterPort}
```

Comment:
It would be nice if we had a derived store with the meaning of this like enabledRemoteDataEncoder

Reply:
Co-signed.

Reply:
Tri-signed.